CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### `Changed`
+
+- Added RASUSA for downsampling of nanopore or pacbio data.
```

Comment:
Nanopore, PacBio capitalization

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/125/commits/667b67d77d74bffb3d9ccedde33cd0f95ee0a188